### PR TITLE
feat: add voiceover to activity1 game1's continue round slides

### LINKED ISSUE #85: 🚀 Feature: Add voiceover for A1G1 middle
this slide: image

we can use this soundtrack: `[SoundTrack.Activity1_G1_Center] : A1_G1_Center,`

### Proposed Solution

* [ ] :wave: I may be able to implement this feature request

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by reginawang99**
closes #85

### CODE REVIEW

File: src/components/Activities/Activity1/Game1/index.tsx [hunk 1]
```diff
@@ -41,12 +44,11 @@ function CipherGame(): JSX.Element {
...
-      return;
-    }
-    setShowSuccess(true);
-    setNumStars(numStars+1);
+    context.next();
+  };
+
+  const continueToGame = () => {
+    context.next();
   };
```
Comment (tr89on) [inline src/components/Activities/Activity1/Game1/index.tsx:52]: what's the difference between advanceGame and continueToGame?

File: src/components/Activities/Activity2/IntroSlides.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ const IntroSlides = [
     child:
     <>
       <LottieControl animationData={A2Animation7.default} />
-      <h2 id={'body-text'}><b>Decompressing</b> is when your computer uses these instructions to show you the video.</h2>
+      <h2 id={'body-text'}><b>Decompressiong</b> is when your computer uses these instructions to show you the video.</h2>
```
Comment (tr89on) [inline src/components/Activities/Activity2/IntroSlides.tsx:58]:
```suggestion
      <h2 id={'body-text'}><b>Decompressing</b> is when your computer uses these instructions to show you the video.</h2>
```

**Review by tr89on — COMMENTED** [review]
reviewed! (some nits)

File: src/components/Activities/Activity1/Game1/index.tsx (hunk 1 above)
Comment (reginawang99) [reply src/components/Activities/Activity1/Game1/index.tsx:52]: yee thats super confusing, ill change it!

File: src/components/Activities/Activity2/IntroSlides.tsx (hunk 2 above)
Comment (reginawang99) [reply src/components/Activities/Activity2/IntroSlides.tsx:58]: nice catch!

**Review by tr89on — APPROVED** [review]
LGTM

**Pull request merged**